components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1195,6 +1195,14 @@ func (p *pg) ListRulesForProject(ctx context.Context, projectID string) ([]*v2.R
 		return nil, err
 	}
 
+	// verify project exists, otherwise we just return an empty list
+	// that could be misleading
+	var project v2.Project
+	row := p.db.QueryRowContext(ctx, `SELECT query_project($1, $2)`, projectID, pq.Array(projectsFilter))
```

Comment:
[nit] I know we use backticks everywhere, but we really don't need them 😄 They're convenient when you want `"` in your string, but we don't here 😉 
```suggestion
	row := p.db.QueryRowContext(ctx, "SELECT query_project($1, $2)", projectID, pq.Array(projectsFilter))
```